### PR TITLE
Issue #37: Validator profile page

### PR DESCRIPTION
**Pull request by farebord**
This PR adds a node page to the community site. It can be accessed by clicking on a node name on the Delegations page.

### CODE REVIEW

File: services/community-site/src/pages/Delegation/NodePage.tsx [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+      setDelegations(data.response.data);
+      setDelegationCount(data.response.count);
+    }
+  }, [nodeId, delegationAtTop, delegationPage]);
+
+  useEffect(() => {
+    getBalances();
+    fetchNode();
+    fetchDelegators();
+  }, [fetchNode, fetchDelegators]);
```
Comment (rattrap) [inline services/community-site/src/pages/Delegation/NodePage.tsx:75]: Maybe we should also add `getBalances` to the dep list so we don't get a warning.

File: services/community-site/src/pages/Delegation/NodePage.tsx [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+  const delegationTotalPages = Math.ceil(delegationCount / 20);
+  const offsetIndex = delegationPage === 1 ? 0 : 20 * (delegationPage - 1);
+
+  if (!node) {
+    return null;
+  }
+  return (
+    <div className="runnode">
+      {delegateToNode && (
+        <Modal
```
Comment (rattrap) [inline services/community-site/src/pages/Delegation/NodePage.tsx:91]: I think we should create small components for these modals so we don't copy them everywhere we need delegation and undelegation.

File: services/community-site/src/pages/Delegation/NodePage.tsx [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+                <div className="taraContainerAmount">
+                  <div className="taraContainerAmountTotal">{node?.totalDelegation}</div>
+                  <div className="taraContainerUnit">TARA</div>
+                </div>
+                <div className="taraContainerAmountDescription">Total delegated</div>
+              </div>
+            </div>
+            <div className="nodeDelegationSplit">
+              <div className="barFlex">
+                <div
```
Comment (rattrap) [inline services/community-site/src/pages/Delegation/NodePage.tsx:199]: Maybe we can create a small component for this?

File: services/community-site/src/pages/Delegation/NodePage.tsx [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+        <div className="delegatorsHeader">
+          <span className="delegatorsLegend">Delegators</span>
+          <div className="showOwnDelegation">
+            <Checkbox
+              value={delegationAtTop}
+              onChange={(e) => setDelegationAtTop(e.target.checked)}
+            />
+            Show my delegation at the top
+          </div>
+          <div className="delegatorsPagination">
```
Comment (rattrap) [inline services/community-site/src/pages/Delegation/NodePage.tsx:275]: We definitely have pagination like this in other pages.

File: services/community-site/src/pages/Delegation/NodePage.tsx [hunk 5]
```diff
@@ -0,0 +1,313 @@
...
+            />
+          </div>
+        </div>
+        <div className="tableHeader">
+          <span>Address</span>
+          <span>Amount of TARA delegated</span>
+        </div>
+        <div className="delegators">
+          {delegations.map((delegator, dIndex) => (
+            <div key={dIndex} className="delegatorRow">
```
Comment (rattrap) [inline services/community-site/src/pages/Delegation/NodePage.tsx:296]: Should use `id` as an index.

File: services/community-site/src/pages/Delegation/NodePage.tsx [hunk 6]
```diff
@@ -0,0 +1,313 @@
...
+              <div className="address">
+                <span>{dIndex + 1 + offsetIndex}.</span> {delegator.address}
+              </div>
+              <div className="badges">
+                {delegator.isSelfDelegation && <div className="selfStake">self-stake</div>}
+                {delegator.isOwnDelegation && <div className="ownStake">your delegation</div>}
+              </div>
+              <div className="amount">{delegator.value}</div>
```
Comment (rattrap) [inline services/community-site/src/pages/Delegation/NodePage.tsx:304]: Add `TARA` after the amount

Comment (cblage) [reply services/community-site/src/pages/Delegation/NodePage.tsx:304]: We should also use `ethers.utils.commify` on the value I think

File: services/community-site/src/pages/Delegation/nodePage.scss [hunk 7]
```diff
@@ -0,0 +1,414 @@
+.nodeInfoWrapper {
```
[the remark this answers is not in the document]
Comment (farebord) [reply services/community-site/src/pages/Delegation/nodePage.scss]: Could we just ignore this one? I wrote all in camelCase 😓

[the remark this answers is not in the document]
Comment (cblage) [reply services/community-site/src/pages/Delegation/nodePage.scss]: I meant just the file name, not the classes 😅 , ie just renaming `nodePage.scss` to `node-page.scss` - what do you think?

File: services/community-site/src/pages/Delegation/NodeProfilePage.tsx [hunk 8]
```diff
@@ -0,0 +1,329 @@
...
+              <>
+                <div className="nodeInfoTitle">node operator description</div>
+                <div className="nodeInfoContent">{node?.profile?.description}</div>
+              </>
+            )}
+            {node?.profile?.website && (
+              <>
+                <div className="nodeInfoTitle">node operator Website</div>
+                <div className="nodeInfoContent">
+                  <a href={node?.profile?.website}>{node?.profile?.website}</a>
```
Comment (cblage) [inline services/community-site/src/pages/Delegation/NodeProfilePage.tsx:203]: We should add a `rel="nofollow"` in this href I think

**Pull request merged**